Source/WebCore/layout/formattingContexts/inline/InlineContentConstrainer.h

```diff
@@ -62,37 +84,16 @@ class InlineContentConstrainer {
     const HorizontalConstraints& m_horizontalConstraints;
 
     Vector<InlineItemRange> m_originalLineInlineItemRanges;
-    Vector<float> m_originalLineWidths;
+    Vector<LayoutUnit> m_originalLineWidths;
```

Comment:
I think this is actually a set of line constraints and not line widths. if they were actual widths of lines we should be using InlineLayoutUnits (same applies below to m_maximumLineWidth)